### DISCUSSION TITLE
Dragdrop 拖拽组件疑似bug

### DISCUSSION THREAD

**Discussion by Julinear** [DevCloudFE/vue-devui#163] [General]
Drop Area中元素拖入Drop Area with Sortable中失效
![image](https://user-images.githubusercontent.com/33052315/149687768-1970ef1d-394d-41ef-8e9c-dc551b9320cd.png)

**Comment by kagol** [comment]
@Julinear 感谢反馈，这确实是个bug，转成issue了：[https://github.com/DevCloudFE/vue-devui/issues/187](https://github.com/DevCloudFE/vue-devui/issues/187)

**Comment by kagol** [comment]
@Julinear Fixed by @asdlml6 on #1144
